eva/udfs/abstract/hf_abstract_udf.py

```diff
@@ -43,12 +43,21 @@ def name(self) -> str:
 
     def __init__(self, udf_obj: UdfCatalogEntry, device: int = -1, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        pipeline_args = {entry.key: entry.value for entry in udf_obj.metadata}
+        pipeline_args = self.default_pipeline_args
+        for entry in udf_obj.metadata:
+            if entry.value.isnumeric():
+                pipeline_args[entry.key] = int(entry.value)
+            else:
+                pipeline_args[entry.key] = entry.value
         self.hf_udf_obj = pipeline(**pipeline_args, device=device)
 
     def setup(self, *args, **kwargs) -> None:
         super().setup(*args, **kwargs)
 
+    @property
+    def default_pipeline_args(self) -> dict:
+        return {}
```

Comment:
I am guessing this is for future when we would some default arguments. In that case, a comment here would be nice. Or else it's like a property that is always {}

Reply:
Correct. I actually added default args for the ASR model